### ISSUE TITLE
Canceling my Friendship request

### ISSUE THREAD

**Issue by mughalles** [hootlex/laravel-friendships#97]
Hallo,
first of all thank you for your niche package. What I missed is a function so I can cancel my requests myself. So like 

if($me->hasSentFriendRequestTo($recipient)){
    $me->**cancelSentFriendrequestTo($recipient);**
}

thanks

**Comment by hootlex** [comment]
Great idea @mughalles. Can you submit a PR? 🙂

**Comment by heidilux** [comment]
what about...
```
if ($me->hasSentFriendRequestTo($recipient) {
    $recipient->denyFriendRequest($me);
}
```
Otherwise, you'd have to get into adding a new Status.

Just a thought :)

**Comment by jaspervanbrian** [comment]
Yep that's great, but I think we should implement a dedicated function for cancelling `user's` request, so if the user of this library needs to fire off custom events after cancelling request, he can distinguish whether a user has cancelled his request or the recipient really did deny the friend request. But this solution is nice tho.

**Comment by cesarlanz** [comment]
Greetings, thanks for your package, I would like to know if there is already a solution to "cancel a friend request"?
